docs/docker/ocpdiag.docker

```diff
@@ -0,0 +1,38 @@
+FROM centos:latest
+
+###############################################################
+# 
+# Sample docker build file to experiment with ocpdiag
+#
+# paste this in ocpdiag.docker
+#
+# docker build --rm ocpdiag.docker centos:ocpdiag
+# 
+# docker run -it --rm centos:ocpdiag
+#
+###############################################################
+
+RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
```

Comment:
comments on these? why are they removed?

Reply:
this tweak is needed to circumvent docker bailing out at build time

Reply:
ok but why?
what i was saying was that it's not obvious why you do this, so it would be useful to add a comment in the docker file explaining

Reply:
I hear you and will add a comment if we decide to move this dockerfile forward. The reason is that centos 8 EOL'd and the mirrors were moved to vault.centos.org.